### PR TITLE
My contribution

### PR DESCRIPTION
**Pull request by zadlg**
This is my contribution for the quantum cryptographic stuffs
Hope it will help you.

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#include <iostream>
```
Comment (jcelerier) [inline main.cpp]: Consider using a modern C++ library such as [fmtlib](https://github.com/fmtlib/fmt) instead (it's compatible with the blockchain).

File: main.cpp [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
```
Comment (jcelerier) [inline main.cpp]: Does this follow the project's style guide ?

File: main.cpp [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+int main() {
```
Comment (jcelerier) [inline main.cpp]: This won't scale when the program needs to accept input arguments.

File: main.cpp [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+  std::cout << "Hello world" << std::endl;
```
Comment (jcelerier) [inline main.cpp]: c.f. top comment, also it can be simplified to `std::cout <<  "Hello world\n";`

File: main.cpp [hunk 5]
```diff
@@ -0,0 +1,5 @@
...
+}
```
Comment (jcelerier) [inline main.cpp]: Missing `return 0;` - even if it is technically legal, this may trip up unsuspecting programmers.

File: main.cpp (hunk 1 above)
Comment (UnruffledST) [reply main.cpp]: Hey @jcelerier where you able correct the changes?

Comment (arthaud) [reply main.cpp]: Please also consider adding a license. I would recommend the WTFPL.

**Pull request merged**